docs/enterprise-setup/upgrading-from-community.md

```diff
@@ -0,0 +1,101 @@
+# Existing Instance Upgrades
+
+This page supplements the [Self-Managed Enterprise implementation guide](./implementation-guide.md). It highlights the steps to take if you are currently using Airbyte Self-Managed Community, our free open source offering, and are ready to upgrade to [Airbyte Self-Managed Enterprise](./README.md). 
+
+A valid license key is required to get started with Airbyte Enterprise. [Talk to sales](https://airbyte.com/company/talk-to-sales) to receive your license key. 
+
+These instructions are for you if:
+* You want your Self-Managed Enterprise instance to inherit state from your existing deployment.
+* You are currently deploying Airbyte on Kubernetes.
+* You are comfortable with an in-place upgrade. This guide does not dual-write to a new Airbyte deployment.
+
+### Step 1: Update Airbyte Open Source
+
+You must first update to the latest Open Source community release. We assume you are running the following steps from the root of the `airbytehq/airbyte-platform` cloned repo.
```

Comment:
@colesnodgrass A lot of folks on older versions will have cloned before we did the repo split. How would these instructions look different?

Reply:
@Hesperide Is this covered by our [Upgrading on K8s (0.26.4-alpha and below)](https://docs.airbyte.com/operator-guides/upgrading-airbyte#upgrading-on-k8s-0264-alpha-and-below) docs?

Reply:
I'm not sure, did the repo split coincide with 0.26.4?

Reply:
I'm pretty sure the repo split happened much later

Reply:
As long as our helm-charts were not renamed (during the split).  I think the only difference would be that they need to clone the `airbyte-platform` repo (into a different directory) instead of using their previously cloned `airbyte` repo.  

I'm surprised we don't have existing documentation regarding how to handle this situation from when we first rolled out the split.